jsonToPrometheus/validatorStatisticsToPrometheus/main.go

```diff
@@ -0,0 +1,220 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"syscall"
+	"time"
+
+	"github.com/gin-contrib/cors"
+	"github.com/gin-gonic/gin"
+	"github.com/multiversx/mx-chain-core-go/core"
+	"github.com/multiversx/mx-chain-core-go/core/check"
+	"github.com/multiversx/mx-chain-go/cmd/node/factory"
+	"github.com/multiversx/mx-chain-go/facade"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-chain-logger-go/file"
+	"github.com/multiversx/mx-chain-tools-go/jsonToPrometheus/collector"
+	"github.com/multiversx/mx-chain-tools-go/jsonToPrometheus/config"
+	"github.com/multiversx/mx-chain-tools-go/jsonToPrometheus/httpClientWrapper"
+	"github.com/multiversx/mx-sdk-go/core/http"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/urfave/cli"
+)
+
+var (
+	helpTemplate = `NAME:
+   {{.Name}} - {{.Usage}}
+USAGE:
+   {{.HelpName}} {{if .VisibleFlags}}[global options]{{end}}
+   {{if len .Authors}}
+AUTHOR:
+   {{range .Authors}}{{ . }}{{end}}
+   {{end}}{{if .Commands}}
+GLOBAL OPTIONS:
+   {{range .VisibleFlags}}{{.}}
+   {{end}}
+VERSION:
+   {{.Version}}
+   {{end}}
+`
+	// restApiInterfaceFlag defines a flag for the interface on which the rest API will try to bind with
+	restApiInterfaceFlag = cli.StringFlag{
+		Name: "rest-api-interface",
+		Usage: "The interface `address and port` to which the REST API will attempt to bind. " +
+			"To bind to all available interfaces, set this flag to :8080",
+		Value: facade.DefaultRestInterface,
+	}
+	// proxyURL defines a flag for the proxy url that will be used to fetch validator statistics
+	proxyURL = cli.StringFlag{
+		Name:  "proxy-url",
```

Comment:
this is the api url, actually. Not the proxy (gateway) :)

Reply:
fixed